src/scrapers/movie/IMDB.cpp

```diff
@@ -115,6 +116,15 @@ void IMDB::search(QString searchStr)
         new NetworkReplyWatcher(this, reply);
         connect(reply, &QNetworkReply::finished, this, &IMDB::onSearchIdFinished);
 
+    } else if (QVariant(includeAdult).toBool()) {
```

Comment:
Not necessary if `includeAdult` was a boolean. :-)

Reply:
Thank you for your bool suggestion. Tested and included (ce0587b).